ffmpeg/tests/test_ffmpeg.py

```diff
@@ -782,3 +783,7 @@ def test__multi_output_edge_label_order():
     out1, out2 = get_filter_complex_outputs(flt_cmpl, 'scale2ref')
     assert out1 == get_filter_complex_input(flt_cmpl, 'scale')
     assert out2 == get_filter_complex_input(flt_cmpl, 'hflip')
+
+
+if sys.version_info >= (3, 4):
+    from ._test_asyncio import test_run_asyncio
```

Comment:
Any reason this can't just use pytest's [`skipif`](https://docs.pytest.org/en/documentation-restructure/how-to/skipping.html#id1) instead of doing a conditional import? i.e.
```python
@pytest.mark.skipif(sys.version_info < (3,4), reason='requires python 3.4 or later')
def test_run_asyncio():
    ...
```

Reply:
There is a `yield from` expression, so trying to put it here will make interpreter raise `SyntaxError`